package.json

```
@@ -47,7 +47,7 @@
     "mini-css-extract-plugin": "2.7.7",
     "node-forge": "1.3.1",
     "npm-run-all": "4.1.5",
-    "prettier": "3.2.2",
+    "prettier": "^3.2.4",
```

```suggestion
    "prettier": "3.2.4",
```

Our standard is to use exact versions in package.json - and in fact you'll just end up with another PR from renovate to "pin" this version if you merge this.